sunpy/tests/helpers.py

```diff
@@ -104,3 +104,36 @@ def wrapper(*args, **kwargs):
                                'New image generated and placed at {}'.format(result_image_loc))
 
     return wrapper
+
+
+def _patch_coverage(testdir, sourcedir):
+    import coverage
```

Comment:
It would be good to have some kind of docstring or comment that explains in what way the coverage file will be patched.